packages/ui/src/views/platform/entity-reference-list.tsx

```diff
@@ -139,8 +120,8 @@ export function EntityReferenceList<T extends BaseEntityProps, S = string, F = s
             })}
           </>
         ) : (
-          <StackedList.Item>
-            <StackedList.Field title={apiError || 'No entities found'} />
+          <StackedList.Item disableHover>
+            <StackedList.Field title="No entities found" titleColor="foreground-3" />
```

Comment:
why is apiError section removed? afair @sans-harness added this to be able to support api failure messages